### PR TITLE
Jim/alias table

### PR DESCRIPTION
**Pull request by JimLiu0**

### CODE REVIEW

File: src/leaderboardBot.py [hunk 1]
```diff
@@ -304,3 +302,22 @@ def clearDailyTable(self):
...
+            item = {
+                'Alias': key,
+                'PlayerName': default_alias[key]
+            }
+            self.alias_table.put_item(Item=item)
+
+    def addChannels(self):
+        for key in default_channels:
+            item = {
+                'ChannelName': key
```
Comment (quinnabrvau) [inline src/leaderboardBot.py]: need `'PlayerName'` in item

Comment (quinnabrvau) [reply src/leaderboardBot.py]: or you could add all the channels as aliases also

File: lambda-loader/template.yaml [hunk 2]
```diff
@@ -111,7 +111,38 @@ Resources:
           AttributeName: "Region"
           KeyType: "RANGE" # Sort Key
       TableName: yesterday-rating-record-table
-      
+  ChannelTable:
```
Comment (quinnabrvau) [inline lambda-loader/template.yaml:114]: What is the advantage of making the table with the `templete.yml` instead of from the python interface?

Comment (JimLiu0) [reply lambda-loader/template.yaml:114]: Separation of code. All the infrastructure can be found in template.yml and the business logic in the python file.

File: src/leaderboardBot.py (hunk 1 above)
Comment (JimLiu0) [reply src/leaderboardBot.py]: > need `'PlayerName'` in item

Good call, I think no argument commands aren't supported right now

File: lambda-loader/template.yaml (hunk 2 above)
Comment (JimLiu0) [reply lambda-loader/template.yaml:114]: Do you need python code to create the table locally?

**Pull request merged**